ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -226,74 +244,52 @@
 {% include 'eos/policy-maps-pbr.j2' %}
 {# QOS policy-map #}
 {% include 'eos/policy-maps-qos.j2' %}
-{# ARP #}
-{% include 'eos/arp.j2' %}
+{# ip radius source interfaces #}
+{% include 'eos/ip-radius-source-interfaces.j2' %}
+{# roles #}
+{% include 'eos/roles.j2' %}
 {# route-maps #}
 {% include 'eos/route-maps.j2' %}
-{# router bfd #}
-{% include 'eos/router-bfd.j2' %}
 {# peer-filters #}
 {% include 'eos/peer-filters.j2' %}
+{# router bfd #}
+{% include 'eos/router-bfd.j2' %}
 {# router bgp configuration #}
 {% include 'eos/router-bgp.j2' %}
-{# router igmp configuration #}
-{% include 'eos/router-igmp.j2' %}
-{# router multicast configuration #}
-{% include 'eos/router-multicast.j2' %}
 {# router general configuration #}
 {% include 'eos/router-general.j2' %}
 {# router traffic-engineering configuration #}
 {% include 'eos/router-traffic-engineering.j2' %}
+{# router igmp configuration #}
+{% include 'eos/router-igmp.j2' %}
+{# router isis configuration #}
+{% include 'eos/router-isis.j2' %}
+{# router multicast configuration #}
+{% include 'eos/router-multicast.j2' %}
 {# router ospf configuration #}
 {% include 'eos/router-ospf.j2' %}
 {# router pim sparse mode configuration #}
 {% include 'eos/router-pim-sparse-mode.j2' %}
-{# router isis configuration #}
-{% include 'eos/router-isis.j2' %}
 {# router msdp #}
 {% include 'eos/router-msdp.j2' %}
-{# MPLS #}
-{% include 'eos/mpls.j2' %}
-{# Patch Panel #}
-{% include 'eos/patch-panel.j2' %}
-{# queue monitor streaming #}
-{% include 'eos/queue-monitor-streaming.j2' %}
 {# ip tacacs+ source interfaces #}
 {% include 'eos/ip-tacacs-source-interfaces.j2' %}
-{# ip radius source interfaces #}
-{% include 'eos/ip-radius-source-interfaces.j2' %}
-{# vmtracer sessions #}
-{% include 'eos/vmtracer-sessions.j2' %}
 {# traffic policies #}
 {% include 'eos/traffic-policies.j2' %}
-{# banner #}
-{% include 'eos/banners.j2' %}
-{# ip http client source interfaces #}
-{% include 'eos/ip-http-client-source-interfaces.j2' %}
-{# ip ssh client source interfaces #}
-{% include 'eos/ip-ssh-client-source-interfaces.j2' %}
```

Comment:
After reverting the requence changes, you can replace these two includes with including your new template. That will keep the order for existing deployments.

Reply:
Fixed.